packages/fixie/src/agent.ts

```diff
@@ -118,6 +121,7 @@ export class FixieAgent {
       name: result.data.agent.name,
       description: result.data.agent.description,
       moreInfoUrl: result.data.agent.moreInfoUrl,
+      published: result.data.agent.published,
```

Comment:
```suggestion
..._.pick(result.data.agent, 'name', 'description', 'moreInfoUrl', 'published')
```

minor, in case it feels cleaner to you